helix-term/src/commands.rs

```diff
@@ -160,21 +168,38 @@ macro_rules! commands {
     }
 }
 
-impl Command {
+impl MappableCommand {
     pub fn execute(&self, cx: &mut Context) {
-        (self.fun)(cx);
+        match &self {
+            MappableCommand::Typable { name, args, doc: _ } => {
+                let args: Vec<&str> = args.iter().map(|arg| arg.as_str()).collect();
+                if let Some(command) = cmd::COMMANDS.get(name.as_str()) {
```

Comment:
`COMMANDS` is a confusing name now with typable command and static commands, renaming it would be nice (`TYPABLE_COMMAND_LIST` is already taken though).